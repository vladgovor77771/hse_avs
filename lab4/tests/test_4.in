5
abcde
1
1
bcdef
1
1
cdefg
1
1
defgh
1
1
efghi
1
1
