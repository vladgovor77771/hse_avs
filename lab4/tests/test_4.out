abcde => bcdef
bcdef => cdefg
cdefg => defgh
defgh => efghi
efghi => fghij
